arcgis-ios-sdk-samples/Route and directions/Display route layer/README.metadata.json

```
@@ -0,0 +1,27 @@
+{
+    "category": "Route and directions",
+    "description": "Display a route layer and its directions using feature collection.",
+    "ignore": false,
+    "images": [
+        "display-route-layer.png"
+    ],
+    "keywords": [
+        "directions",
+        "feature collection",
+        "route layer",
+        "AGSFeatureCollection",
+        "AGSFeatureCollectionLayer",
+        "AGSFeatureCollectionTable"
+    ],
+    "redirect_from": [],
+    "relevant_apis": [
+        "AGSFeatureCollection",
+        "AGSFeatureCollectionLayer",
+        "AGSFeatureCollectionTable"
+    ],
+    "snippets": [
+        "DisplayDirectionsViewController.swift",
+        "DisplayRouteLayerViewController.swift"
```

```suggestion
        "DisplayRouteLayerViewController.swift",
        "DisplayDirectionsViewController.swift"
```
So that they are the right order on developer website.